libs/sdk-ui-pivot/src/impl/stickyRowHandler.ts

```diff
@@ -49,7 +50,16 @@ function shouldUpdate(
     return initialUpdate || differentRow || differentHorizontalBreakpoint;
 }
 
-export const updateStickyRowContentClasses = (
+const areDataDifferent = (previousData: any, currentData: any): boolean => {
+    return (
+        Object.keys(previousData).length !== Object.keys(currentData).length ||
+        Object.keys(previousData).some((dataItemKey: string) => {
```

Comment:
Are we ok if this returns true for data that is effectively the same, just not referentially identical? Or should we use deep equality?

Reply:
data in this case are strings, so "return previousData[dataItemKey] !== currentData[dataItemKey]"  should be enough

Reply:
@ivan-nejezchleb you are expert in the sticky row handler now, do you have some insight into this? :)

Reply:
ok, makes sense, thanks @ivan-nejezchleb, even though lodash.isEqual would be effectively the same here, right?